app/views/onboarding/Onboarding4.js

```diff
@@ -17,7 +18,10 @@ const Onboarding4 = (props) => {
   const handleOnPressNext = () =>
```

Comment:
could we avoid the nested ternaries with something like
```suggestion
  const handleOnPressNext = () => {
     const nextScreen = () => {
       if (isGPS && isPlatformiOS()) {
         return Screens.OnboardingNotificationPermissions
       } else if (isGPS) {
         return Screens.OnboardingLocationPermission
        } else {
        Screens.EnableExposureNotifications
        }
        props.navigation.replace(nextScreen()
        }
     
```